package.json

```diff
@@ -12,6 +12,6 @@
   },
   "version": "0.1.0",
   "devDependencies": {
-    "webpack-dev-server": "^4.0.0"
+    "webpack-dev-server": "^4.1.0"
```

Comment:
This is not part of the ticket's description, so it would've been better to have its own PR for it or at least its own commit explaining you're upgrading it. Anyway, it's always good to upgrade, just be careful not to group unrelated changes into the same commit in the future. 👍